src/ofdm.c

```diff
@@ -167,13 +167,10 @@ struct OFDM *ofdm_create(const struct OFDM_CONFIG *config) {
     float tval;
     int i, j;
 
-    /* Check if called correctly */
+    ofdm = (struct OFDM *) MALLOC(sizeof (struct OFDM));
+    assert(ofdm != NULL);
 
     if (config == NULL) {
-        return NULL;
-    }
-
-    if (config->nc == 0) {
         /* Fill in default values */
```

Comment:
This is the other change to the modem API - I felt this was more intuitive, and it simplifies the ```ofdm_create()``` calls